.github/workflows/sync.yml

```diff
@@ -0,0 +1,40 @@
+name: Upstream Sync
```

Comment:
This file is only for WTFAcademy. Could you change your base branch on the AmazingAng:main. Not merge the WTFAcademy fork's repo.

Reply:
Or you can delete this file and commit it again.

Reply:
I deleted the file, and in the future I will fork the AmazingAng:main repo to work based on that fork